Controllers/conversationController.js

```diff
@@ -0,0 +1,141 @@
+const CatchAsyncError = require('../Utils/CatchAsyncError')
+const db = require('../db/models/index')
+const User = db.User
+const UserConversation = db.UserConversation
+const Messages = db.Message
+const { Op } = require('sequelize')
+
+// WORK IN PROGRESS
+exports.createConversation = CatchAsyncError(async (req, res, next) => {
+  const searchedUserUsername = req.body.searchedUserUsername
+  const currentUserUsername = req.body.currentUserUsername
```

Comment:
Getting the current user should be done by reading the requests JWT. It's best if you create a helper function that does this.